dev/manage.py

```diff
@@ -1,42 +1,51 @@
+from asyncio.log import logger
```

Comment:
Minor: It looks like `logger` is not being used here, so the import can probably be removed.